hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -75,7 +75,10 @@ module Hedgehog.Internal.Property (
   , eval
   , evalM
   , evalIO
+  , evalMaybe
+  , evalMaybeM
   , evalEither
+  , evalEitherM
```

Comment:
It'd be nice to have those in alphabetical order, after `evalEitherXyz`.

Reply:
sorted evalMaybeXyz after evalEither